src/modules/tests/journal-data/common/candidate/candidate.action-creator.ts

```diff
@@ -26,6 +26,9 @@ export const createPopulateCandidateDetailsAction = (testCategory: string, booki
         businessName: get(booking, 'business.businessName'),
         businessTelephone: get(booking, 'business.telephone'),
       });
+    case TestCategory.EUA1M2:
```

Comment:
just add the category AM2s to the TestCategory.B as they will be used from the common